src/handlers/peer-events.js

```diff
@@ -207,6 +211,10 @@ export const handleCowTradeRequestAccept = (farmhand, cowReceived, peerId) => {
         'success'
       )
 
+      if (!didOriginallyOwnReceivedCow) {
```

Comment:
I added this to prevent the player from receiving experience by re-trading their own cows.

Reply:
good call!